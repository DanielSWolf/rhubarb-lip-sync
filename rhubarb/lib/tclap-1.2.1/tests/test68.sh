#!/bin/sh

# this tests whether we can parse args from
# a vector of strings and that combined switch
# handling doesn't get fooled if the delimiter
# is in the string
# success  
../examples/test13 > tmp.out 2>&1

if cmp -s tmp.out $srcdir/test68.out; then
	exit 0
else
	exit 1
fi

