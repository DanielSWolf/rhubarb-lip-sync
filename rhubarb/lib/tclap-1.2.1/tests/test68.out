module
MultiSwtichArg was found 0 times.
done...
